Jropp/jason-ropp-js-iobp
# Stylesheet

## What It Does

Adds external CSS stylesheet to the project. 

Previously an external stylesheet was left out because of the complications of doing so with webpack. I'm now circling back to add that in.

This required some changes in webpack:

**webpack.config.js**
```js
{
          test: /\.html$/,
          use: [
            { loader: "babel-loader" },
            {
              options: {
                 // This option changes links to css into require statements to be processed by the css-load 
                later
                processStyleLinks: true
              },
              loader: "polymer-webpack-loader"
            }
          ]
        },
```
and
```js
{
          test: /\.css$/,
          use: [{ loader: "css-loader" }]
        }
```
## How To Test

The CSS styling should be the same as it has recently been. So if you notice any strange or missing styling, then something is probably wrong with the connecting stylesheet.

Also you could change the background color of `.default-card` to be red in styles.css as a check.

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:

## Dependencies
